docs/commands/join.md

```diff
@@ -82,98 +89,89 @@ x-tagGroups:
     description: "Text from info: description of the second input file"
 ```
 
-
-If some operations in an input file don't have a tag assigned to them, the `join` command automatically adds the `other` tag to those operations in the output file. The `other` tag is also included in the `x-tagGroups` object.
-
-If any of the input files contains the `x-tagGroups` object, the content of this object is ignored by the `join` command, and not included in the output file.
+:::info
+* If some operations in an input file don't have a tag assigned to them, the `join` command automatically adds the `other` tag to those operations in the output file. The `other` tag is also included in the `x-tagGroups` object.
+* If any of the input files contain the `x-tagGroups` object, the content of this object is ignored by the `join` command and not included in the output file.
+:::
 
 The `servers` object combines the content from all input files, starting with the content from the first file. Commented lines are not included in the output file.
 
 Path names and component names must be unique in all input files, but their content doesn't have to be unique for the `join` command to produce the output file. For each path item object, only the operation ID must be unique.
 
 If the `join` command detects any conflicting content while trying to combine the input files, it displays informative messages about the conflicts and exits without creating an output file. To prevent this, use optional parameters to add prefixes to tags and components.
 
+```bash example of conflicts
```

Comment:
I really like that you included this example! Good thinking - it will help users recognize this type of error output, so they know what to expect. :brain: